crates/spirv-std/macros/src/image.rs

```diff
@@ -420,6 +435,20 @@ mod params {
     use super::*;
     use proc_macro2::TokenStream;
 
+    pub fn image_access_from_str(s: &str) -> Result<AccessQualifier, &'static str> {
```

Comment:
Any reason to not use TryFrom instead?

Reply:
Just that this is a smaller diff and it follows the style that's already in the module.